cpp/tests/strings/chars_types_tests.cpp

```diff
@@ -22,6 +22,8 @@
 #include <cudf_test/column_utilities.hpp>
 #include <cudf_test/column_wrapper.hpp>
 
+#include <thrust/iterator/transform_iterator.h>
+
```

Comment:
similar includes without any changes are present in many files. check them all at once.

Reply:
No I inserted manually.